src/main/java/github/weichware10/util/db/Configurations.java

```diff
@@ -111,79 +111,68 @@ public Configuration get(String configId) {
      * @return Erstellte ID der hinzugefügten Konfiguration
      */
     public String set(Configuration configuration) {
-        final String ccQueryFormat = """
+        final String queryF = String.format("""
                 INSERT INTO %s.configurations
                 (configid, tooltype, tutorial, question, imageurl,
                 intro, outro,
                 strings, initialsize_x, initialsize_y, timings_0, timings_1,
                 imageview_width, imageview_height, speed)
                 VALUES
-                ('%s', '%s', %d, '%s', '%s',
-                '%s', '%s',
-                '%s', %d, %d, %d, %d,
-                null, null, null);""";
+                (?, ?, ?, ?, ?,
+                ?, ?,
+                ?, ?, ?, ?, ?,
+                ?, ?, ?);""", dataBaseClient.schema);
 
-        final String zmQueryFormat = """
-                INSERT INTO %s.configurations
-                (configid, tooltype, tutorial, question, imageurl,
-                intro, outro,
-                strings, initialsize_x, initialsize_y, timings_0, timings_1,
-                imageview_width, imageview_height, speed)
-                VALUES
-                ('%s', '%s', %d, '%s', '%s',
-                '%s', '%s',
-                null, null, null, null, null,
-                %s, %s, %s);""";
-
-        final String uniqueException =
-                "ERROR: duplicate key value violates unique constraint \"configurations_pkey\"";
-
-        String query = null;
+        final String uniqueException = "ERROR: duplicate key value violates unique constraint \"configurations_pkey\"";
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 119).